urlconnection/src/test/java/zipkin/reporter/urlconnection/URLConnectionSenderTest.java

```diff
@@ -161,6 +183,33 @@ public void illegalToSendWhenClosed() throws Exception {
     send(TestObjects.TRACE);
   }
 
+  @Test
+  public void jsonEncoding() throws Exception {
```

Comment:
If checking json encoding I wouldn't copy paste the assertions that don't have to do with json. Ex just check the sender.encoding and mediatype then send once and check they were received via stats (no need to check pre-send and second send as they don't have anything to do with json). This keeps the code more to the point and.. shorter!